File: Models/Responses/Budgets/ResponseSector.cs
namespace Models.Responses.Budgets;

/// <summary>
/// 섹터 정보 응답 모델 
/// </summary>
public class ResponseSector : ResponseCommonWriter
{
    /// <summary>
    /// 섹터 아이디 
    /// </summary>
    public Guid Id { get; init; }

    /// <summary>
    /// 섹터 값
    /// </summary>
    public string Value { get; init; } = "";
}

File: BudgetBringerV2/src/models/responses/budgets/response-sector.ts
import {ResponseCommonWriter} from "../response-common-writer";

/**
 * 섹터 정보 응답 모델
 */
export class ResponseSector extends ResponseCommonWriter {
  /**
   * 아이디
   */
  id: string;

  /**
   * 섹터 값
   */
  value: string;
}
